### ISSUE TITLE
When the apk is unable to reverse - Download Java option should not be available 

### ISSUE THREAD

**Issue by bugwrangler** [ajinabraham/YSO-Mobile-Security-Framework#20]

**Comment by bugwrangler** [comment]
Ajin - Let me know if you need APK Example - i shall share apk offline.

**Comment by ajinabraham** [comment]
Yes please. So that I can reproduce this bug. Send it to ajin25 [gmail]

**Comment by ajinabraham** [comment]
It's working fine for me. sending you the Report via mail.

**Comment by bugwrangler** [comment]
I will check on latest version and update it

**Comment by bugwrangler** [comment]
Issue replicates on Windows 7 machine, can you please update - any fix available?

**Comment by ajinabraham** [comment]
Looks like in Windows, the internal component dex2jar is having some issues in finding java executable path.

**Comment by ajinabraham** [comment]
A quick fix you can do is to modify the file YSO-Mobile-Security-Framework/StaticAnalyzer/tools/d2j/d2j-dex2jar.bat

and change the line 
```bash
java -Xms512m -Xmx1024m -cp %CLASSPATH% "com.googlecode.dex2jar.tools.Dex2jarCmd" %*
```

with 

```bash
"C:\Program Files\Java\jdk1.7.0_75\bin\java" -Xms512m -Xmx1024m -cp %CLASSPATH% "com.googlecode.dex2jar.tools.Dex2jarCmd" %*
````

provided your java executable is located at "C:\Program Files\Java\jdk1.7.0_75\bin\" directory.

**Comment by bugwrangler** [comment]
Will verify the workaround and update you accordingly

**Comment by ajinabraham** [comment]
That issue is fixed in 0.8beta. Use it instead of work around
